internal/internal_task_handlers_interfaces_test.go

```diff
@@ -243,3 +244,67 @@ func (s *PollLayerInterfacesTestSuite) TestGetNextCommandsSdkFlags() {
 
 	s.Equal(0, len(sdkFlags))
 }
+
+func (s *PollLayerInterfacesTestSuite) TestMessageCommands() {
+	// Schedule an activity and see if we complete workflow.
+	taskQueue := "tq1"
+	testEvents := []*historypb.HistoryEvent{
+		createTestEventWorkflowExecutionStarted(1, &historypb.WorkflowExecutionStartedEventAttributes{TaskQueue: &taskqueuepb.TaskQueue{Name: taskQueue}}),
+		createTestEventWorkflowTaskScheduled(2, &historypb.WorkflowTaskScheduledEventAttributes{TaskQueue: &taskqueuepb.TaskQueue{Name: taskQueue}}),
+		createTestEventWorkflowTaskStarted(3),
+		createTestEventWorkflowTaskCompleted(4, &historypb.WorkflowTaskCompletedEventAttributes{
+			ScheduledEventId: 2,
+			StartedEventId:   3,
+		}),
+		createTestEventWorkflowTaskScheduled(5, &historypb.WorkflowTaskScheduledEventAttributes{TaskQueue: &taskqueuepb.TaskQueue{Name: taskQueue}}),
+		createTestEventWorkflowTaskStarted(6),
+		{
+			EventId:   7,
+			EventType: enumspb.EVENT_TYPE_WORKFLOW_TASK_FAILED,
+		},
+		{
+			EventId:   8,
+			EventType: enumspb.EVENT_TYPE_WORKFLOW_EXECUTION_UPDATE_ACCEPTED,
```

Comment:
How can WFT be failed but also have update accepted?

Reply:
good catch, yeah i am missing an event here